src/Monkeypatcher.cc

```diff
@@ -953,7 +953,48 @@ void patch_after_exec_arch<ARM64Arch>(RecordTask* t, Monkeypatcher& patcher) {
     return;
   }
 
-  FATAL() << "Unimplemented";
+  auto vdso_start = t->vm()->vdso().start();
+
+  VdsoReader reader(t);
+  auto syms = reader.read_symbols(".dynsym", ".dynstr");
+
+  static const named_syscall syscalls_to_monkeypatch[] = {
+#define S(n) { "__kernel_" #n, ARM64Arch::n }
+    S(rt_sigreturn), S(gettimeofday), S(clock_gettime), S(clock_getres),
```

Comment:
I think ideally this would share more code with the x86 implementation but I'm not super concerned about it.